packages/Spellcheck-Core.package/SpellingTextParser.class/instance/addParsedWord.from.to.into..st

```diff
@@ -0,0 +1,17 @@
+as yet unclassified
+addParsedWord: aString from: startIndex to: endIndex into: words
+
+	| camelCaseWords word cc_word |
```

Comment:
can we rename 'cc_word'? It looks a bit odd that we use camelCase for the name of the first variable but underscore for the other variable